### PR TITLE
Misc. Fixes

### PR DESCRIPTION
**Pull request by protich**
Collection of fixes for v1.11.x

### CODE REVIEW

File: include/class.filter.php [hunk 1]
```diff
@@ -369,6 +369,7 @@ function delete() {
         $sql='DELETE FROM '.FILTER_TABLE.' WHERE id='.db_input($id).' LIMIT 1';
         if(db_query($sql) && ($num=db_affected_rows())) {
             db_query('DELETE FROM '.FILTER_RULE_TABLE.' WHERE filter_id='.db_input($id));
+            db_query('DELETE FROM '.FILTER_ACTION_TABLE.' WHERE filter_id='.db_input($id));
```
Comment (aydreeihn) [inline include/class.filter.php]: This works, however, as a result of some of our latest commits, the filter_id on filter actions isn't being set. I'll look into it

File: include/class.format.php [hunk 2]
```diff
@@ -529,6 +529,10 @@ function __formatDate($timestamp, $format, $fromDb, $dayType, $timeType,
         if (!$timestamp || !($datetime = DateTime::createFromFormat('U', $timestamp)))
             return '';
 
+        // Normalize timezone
+        if ($timezone)
+            $timezone = Format::timezone($timezone);
```
Comment (aydreeihn) [inline include/class.format.php]: Where were the timezones invalid before? From what I tested the timezone showed the same before and after this code.

File: include/class.queue.php [hunk 3]
```diff
@@ -2825,7 +2826,8 @@ class DateTimeFilter
     static $desc = /* @trans */ "Date and Time";
 
     function filter($text, $row) {
-        return $text->changeTo(Format::datetime($text->value));
+        return $text ?
```
Comment (aydreeihn) [inline include/class.queue.php]: Should this be $text->value instead of just $text?

Comment (protich) [reply include/class.queue.php]: No text can be null, causing a fatal error.

File: include/class.format.php (hunk 2 above)
Comment (protich) [reply include/class.format.php]: Parsing a timezone from a datetime from database you can end up with a timezone like  'GMT' or '+00:00' - which are not valid. 

Now that you mentioned it, it looks line 538 does the same thing - will drop this change.

**Pull request merged**